provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1692,6 +1692,26 @@ func nodeGroupProperties(cluster, v2 bool) map[string]schema.PropertySpec {
 			TypeSpec:    schema.TypeSpec{Type: "integer"},
 			Description: "The size in GiB of a cluster node's root volume. Defaults to 20.",
 		},
+		"nodeRootVolumeDeleteOnTermination": {
+			TypeSpec:    schema.TypeSpec{Type: "boolean"},
+			Description: "Whether the root block device should be deleted on termination of the instance. Defaults to true.",
+		},
+		"nodeRootVolumeEncrypted": {
+			TypeSpec:    schema.TypeSpec{Type: "boolean"},
+			Description: "Whether to encrypt a cluster node's root volume. Defaults to false.",
+		},
+		"nodeRootVolumeIops": {
+			TypeSpec:    schema.TypeSpec{Type: "integer"},
+			Description: "The amount of provisioned IOPS. This is only valid with a volumeType of 'io1'.",
+		},
+		"nodeRootVolumeThroughput": {
+			TypeSpec:    schema.TypeSpec{Type: "integer"},
```

Comment:
No floats here?

Reply:
Yes, this is an integer only value. Ref: https://docs.aws.amazon.com/cli/latest/reference/emr/create-cluster.html